cmake/EkatUtils.cmake

```diff
@@ -113,3 +113,19 @@ macro (EkatDisableAllWarning targetName)
 
 endmacro (EkatDisableAllWarning)
 
+function(separate_cut_arguments prefix options oneValueArgs multiValueArgs return_varname)
```

Comment:
Since you plan to set stuff in the parent scope, you could use a macro instead of a function, and do away with the `result` temp var.